projects/fsastorefrontstyles/_index.scss

```diff
@@ -5,8 +5,6 @@
 @import 'scss/fs-mixins';
 @import 'scss/overrides';
 @import 'scss/base';
-
-@import 'scss/theme';
```

Comment:
Delete _theme.scss as well, since it is not used.

Reply:
It's needed by the root.scss file. I would leave it, because it's purpose is to easily switch between themes. Granted, we only have one at the moment, but we may have some more in the future.